android-sdk-restclient-test/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,11 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
```

Comment:
You are correct, the colors are defined in the colors.xml.  They are referenced in this file using @color/colorPrimary

---
In reply to: [116610925](https://github.com/Microsoft/healthvault-java-sdk/pull/5#discussion_r116610925) [](ancestors = 116610925)